packages/block-editor/src/components/inner-blocks/button-block-appender.js

```diff
@@ -9,9 +14,13 @@ export const ButtonBlockAppender = ( {
 	showSeparator,
 	isFloating,
 	onAddBlock,
+	isToggle,
 } ) => {
 	return (
 		<BaseButtonBlockAppender
+			className={ classnames( {
+				'block-list-appender__toggle': isToggle,
```

Comment:
There's some code smell with this classname, it was added to `BlockListAppender`, but it should really be part of the `ButtonBlockAppender` component. It's also missing the `block-editor` prefix. I'll work on a separate PR to fix it.